### PR TITLE
Fixed broken link

### PR DESCRIPTION
**Pull request by sangling**
This link is no longer valid
https://learn.microsoft.com/microsoft-365/security/defender-endpoint/microsoft-365/security/defender/configure-email-notifications

Updated to
https://learn.microsoft.com/microsoft-365/security/defender-endpoint/configure-vulnerability-email-notifications

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/endpoint-attack-notifications.md [hunk 1]
```diff
@@ -58,7 +58,10 @@ Endpoint Attack Notifications are alerts that have been hand crafted by Microsof
 - The alerts queue in the Microsoft 365 Defender portal
 - Using the [API](../../security/defender-endpoint/get-alerts.md)
 - [DeviceAlertEvents](../../security/defender-endpoint/advanced-hunting-devicealertevents-table.md) table in Advanced hunting
-- Your email if you [configure an email notifications](../../security/defender-endpoint/configure-email-notifications.md) rule
+- Your email if you [configure an email notifications](../../security/defender-endpoint/configure-vulnerability-email-notifications.md) rule
+
+
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/endpoint-attack-notifications.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sangling, kindly commit the suggested change to remove the extra blank lines. Many thanks.

**Comment by sangling** [comment]
Ahh - so sorry about that - that was sloppy of me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@vpattnai 
Hi,
As author of the article for the proposed change in this PR, do you approve of the change?

Please @ mention me in your response.

Thanks!

**Pull request merged**